### ISSUE TITLE
some question about log

### ISSUE THREAD

**Issue by ysf19959209360** [albrateanu/LYT-Net#28]
Hello author! could you provide the trained log？Thanks！

**Comment by albrateanu** [comment]
Hi @ysf19959209360 

We did not store training logs for the experiments as the framework isn't implemented to do so.

**Comment by ysf19959209360** [comment]
> Hi [@ysf19959209360](https://github.com/ysf19959209360)
> 
> We did not store training logs for the experiments as the framework isn't implemented to do so.

Hello, thank you for your quick response! I have another question: why am I unable to achieve a score of 26 when reproducing the model in PyTorch with your default settings?

**Comment by ysf19959209360** [comment]
Why is the loss value fluctuating rather than gradually converging when training on TensorFlow?

**Comment by ysf19959209360** [comment]
Why is the loss value fluctuating rather than gradually converging when training on TensorFlow?

![Image](https://github.com/user-attachments/assets/bf2a5e20-8f7b-4ec9-9d19-b7beaa8cc409)

**Comment by albrateanu** [comment]
PyTorch isn't stable. There were issues which we have to fixm

The Tensorflow loss is printed at every sample, just for logging. It's not avg. epoch loss.

**Comment by ysf19959209360** [comment]
> The Tensorflow loss is printed at every sample, just for logging. It's not avg. epoch loss.

Thanks! How many epochs does it typically take to get the best model in TensorFlow? I've run 400 epochs and the highest PSNR I got is only 26.15.

![Image](https://github.com/user-attachments/assets/8acdd50c-e927-457f-ba28-cda5bb1d7984)
